src/pods/canvas/canvas-put-table-on-top.business.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { putTableOnTop } from './canvas.business';
+import { TableVm } from './canvas.vm';
+
+describe('putTabbleOnTop', () => {
+  it('should return empty array, if I have a table and the array is empty', () => {
+    // Arrange
+    const tables: TableVm[] = [];
+    const tableId = 'someTableId';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+
+  it('should return the same array, if table and array with one element,but it is not the same as table', () => {
+    // Arrange
+    const tables: TableVm[] = [
+      { id: '1', fields: [], tableName: 'Table 1', x: 0, y: 0 },
+    ];
+    const tableId = '2';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual(tables);
+  });
+
+  it("should return an array with a single table, which is the same table if it's the only one", () => {
+    // Arrange
+    const TableId = '1';
+    const tables: TableVm = {
```

Comment:
SInce this variable is assign to an object instead of an array, it should be named `table`

Could you update it?

```diff

- const tables: TableVm = {
+ const table: TableVm = {

     id: TableId,
      fields: [],
      tableName: 'Table 1',
      x: 0,
      y: 0,
    };
```